bin/tessel-2.js

```diff
@@ -24,6 +24,7 @@ function makeCommand(commandName) {
     .option('key', {
       required: false,
       metavar: 'PRIVATEKEY',
+      default: controller.LOCAL_AUTH_KEY,
```

Comment:
@rwaldron if @johnnyman727 allows me to do :wink: ... I tried to add as less changes and code as I can. But former `var Tessel = require('../lib/tessel/tessel');` was in the `tessel-2.js` and I used `default: Tessel.LOCAL_AUTH_KEY`(PR wasn't merged).
I also would prefer this. Would you agree @johnnyman727 ? My local tests pass and the solution works. Waiting for submitting PR until getting a feedback.